var/spack/repos/builtin/packages/py-pygobject/package.py

```diff
@@ -10,22 +10,38 @@ class PyPygobject(PythonPackage):
     """bindings for the GLib, and GObject,
     to be used in Python."""
 
-    homepage = "https://pypi.python.org/pypi/pygobject"
-
-    version("3.38.0", sha256="0372d1bb9122fc19f500a249b1f38c2bb67485000f5887497b4b205b3e7084d5")
-    version("3.28.3", sha256="3dd3e21015d06e00482ea665fc1733b77e754a6ab656a5db5d7f7bfaf31ad0b0")
-    version("2.28.6", sha256="fb8a1d4f665130a125011659bd347c7339c944232163dbb9a34fd0686577adb8")
+    homepage = "https://pygobject.readthedocs.io/en/latest/"
+
+    version("3.46.0", sha256="426008b2dad548c9af1c7b03b59df0440fde5c33f38fb5406b103a43d653cafc")
+    version("3.38.0",
+            sha256="0372d1bb9122fc19f500a249b1f38c2bb67485000f5887497b4b205b3e7084d5",
+            deprecated=True
+    )
+    version("3.28.3",
+            sha256="3dd3e21015d06e00482ea665fc1733b77e754a6ab656a5db5d7f7bfaf31ad0b0",
+            deprecated=True
+    )
+    version("2.28.6",
+            sha256="fb8a1d4f665130a125011659bd347c7339c944232163dbb9a34fd0686577adb8",
+            deprecated=True
+    )
 
     extends("python")
 
     depends_on("py-setuptools", type="build")
     depends_on("pkgconfig", type="build")
-    depends_on("libffi")
-    depends_on("glib")
-    depends_on("py-pycairo", type=("build", "run"))
-    depends_on("gobject-introspection")
     depends_on("gtkplus")
 
```

Comment:
Needs `python@3.8:3` dep

Reply:
Can you add a comment like:
```python
# meson.build
```
so it's obvious where to look to find dependency versions? Would also help to sort them in the same order.